### ISSUE TITLE
Siren References, Release Notes

### ISSUE THREAD

**Issue by BranchS** [ArtSabintsev/Harpy#194]
In the latest version of harpy, some of the debug printouts accidentally refer to Harpy by its sister's name, Siren. (just search the project for Siren, and you'll find them).

Also, just a request: for an add-on so focused on release notes, it would be nice to have release notes for the new versions of Harpy!

**Comment by ArtSabintsev** [comment]
Hi BranchS,

Would you mind opening a PR that fixes this issue?

Also, SIren supports release notes. Harpy does not at the moment, but I'll add it as soon as I get some time.

**Comment by BranchS** [comment]
Hi! I don't see release notes for Siren either. I don't mean adding support for anything-- just official documentation for when things change with the code would be nice. I'll try to make a PR now (it's my first one, so I may screw it up).

**Comment by ArtSabintsev** [comment]
Harpy's release notes are here: https://github.com/ArtSabintsev/Harpy/releases
Siren's release notes are here: https://github.com/ArtSabintsev/Siren/releases

Sure, good luck on the PR!

**Comment by BranchS** [comment]
So I couldn't figure out how to make a pull, but here's the updated files. You should probably double check things to make sure I didn't break anything (though I didn't change anything that wasn't a comment/text).
[Harpy 4.1.13.zip](https://github.com/ArtSabintsev/Harpy/files/1965260/Harpy.4.1.13.zip)

are any plans to fix the "Infinite Loop Updates" in Harpy? for those of us using Obj-C, we can't move to Siren.

**Comment by ArtSabintsev** [comment]
@BranchS Don't worry about the PR. I'll address it this week. Thanks for the attempt!

As for the infinite loop, that should be addressed by setting this variable `setShowAlertAfterCurrentVersionHasBeenReleasedForDays`. 

And as an Objective-C user, you can still use Swift libraries: https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html

**Comment by BranchS** [comment]
Oh cool! Thanks. Then the issue templates should probably be updated to reflect that.

Thanks again.

**Comment by ArtSabintsev** [comment]
Good idea. Will do that as well when I get to fixing this existing issue.

**Comment by BranchS** [comment]
The issue template isn’t fixed in github. When a new issue is created, it still has the old text...

**Comment by ArtSabintsev** [comment]
It's fixed. Common Issue #4 is what I added. Maybe refresh your cache?

**Comment by BranchS** [comment]
I see common issue number 4, but it still links to Siren instead of suggesting the Harpy solution. Either way is fine I suppose. Feel free to close this.

**Comment by ArtSabintsev** [comment]
The solution exists in Harpy as well. I reference the iVar in Harpy in the issue.

**Comment by ArtSabintsev** [comment]
oh, stupid me. I see what you mean. I blindly copied pasted. Long day. Sorry about that.

**Comment by ArtSabintsev** [comment]
ok, fixed for real - thanks for your vigilance!
